src/components/network/NewNetwork.tsx

```diff
@@ -147,7 +156,12 @@ const NewNetwork: React.SFC = () => {
             </Col>
           </Row>
           <Form.Item>
-            <Button type="primary" htmlType="submit" loading={createAsync.loading}>
+            <Button
+              type="primary"
+              htmlType="submit"
+              loading={createAsync.loading}
+              disabled={!isDockerNetworkNameValid}
```

Comment:
It's become a bit of an anti-pattern to disable buttons based on validation states because it's not always obvious why the button is disabled. It's better to let the user click it and display an error, similar to how the Name field works.